views/res/i18n/de.ini

```diff
@@ -0,0 +1,268 @@
+OPTION_SCREEN_DPI_OVERRIDE_NONE=Standard Bildschirm DPI
+OPTION_GDC_EXECUTABLE=GDC-Anwendung
+OPTION_LANGUAGE=Sprache
+OPTION_LDC2_EXECUTABLE=LDC2-Anwendung
+OPTION_LDMD2_EXECUTABLE=LDMD2-Anwendung
+OPTION_MODULE_NAME=Modulname
+OPTION_NEW_PROJECT=Neues Projekt
+OPTION_NEW_SOURCE_FILE=Neue Quelldatei
+OPTION_NEW_WORKSPACE=Neue Workspace
+OPTION_PROJECT_NAME=Projektname
+OPTION_PROJECT_TEMPLATE=Projektvorlage
+OPTION_RDMD_ADDITIONAL_PARAMS=rdmd zusätzliche Parameter
+OPTION_RDMD_EXECUTABLE=rdmd-Anwendung
+OPTION_RUN_IN_EXTERNAL_CONSOLE=Starte in externen Konsole
+OPTION_RUN_DEBUG=Starte und debugge
+OPTION_SHOW_SPACES=Zeige Leerzeichenmarkierungen an
+OPTION_SHOW_TABS=Zeige Tabpositionsmarkierungen an
+OPTION_SMART_INDENTS=Schlaues Einrücken
+OPTION_SMART_INDENTS_PASTE=Schlaues Einrücken nach Einfügen
+OPTION_TAB=Tabgröße
+OPTION_TEMPLATE_DESCR=Vorlagenbeschreibung
+OPTION_TEXT_EDITORS=Texteditor
+OPTION_TERMINAL=Terminal
+OPTION_TERMINAL_EXECUTABLE=Terminal-Anwendung
+OPTION_THEME=Theme
+OPTION_TOOLCHAIN=Toolchain
+OPTION_TOOLCHAINS=Toolchains
+OPTION_VERBOSE=Verbose
+OPTION_WORKING_DIR=Arbeitsverzeichnis
+OPTION_WORKSPACE_NAME=Workspace-Name
+OPTION_USE_SPACES=Nutze Leerzeichen für Tabs
+
+ERROR=Fehler
+ERROR_CANNOT_CREATE_PROJECT=Projekte konnte nicht erstellt werden
+ERROR_CANNOT_DEBUG_PROJECT=Projekt konnte nicht gedebuggt werden
+ERROR_CANNOT_FIND_EXEC=Ausführbare Datei nicht gefunden
+ERROR_CANNOT_RUN_PROJECT=Projekt konnte nicht gestartet werden
+ERROR_FAILED_CREATE_PROJECT=Projekt konnte nicht erstllt werden
+ERROR_STARTUP_PROJECT_ABSENT=Kein Startprojekt angegeben
+ERROR_INVALID_NUMBER=Unzulässige Nummer
+ERROR_INVALID_PARAMETERS=Ungültiger Parameter
+ERROR_INVALID_WORKSPACE_FILE=Ungültige Workspace-Datei
+ERROR_INVALID_WS_OR_PROJECT_FILE=Diese Datei ist keine gültige Workspace-/Projektdatei
+ERROR_NO_DEBUGGER=Kein Debugger in den Einstellungen angegeben
+ERROR_NO_SUCH_LINE=Keine solche Zeile
+ERROR_OPEN_FILE=Datei konnte nicht geöffnet werden
+ERROR_OPEN_PROJECT=Projekt konnte nicht geöffnet werden
+ERROR_OPEN_WORKSPACE=Workspace konnte nicht geöffnet werden
+ERROR_OPENING_FILE=Datei konnte nicht geöffnet werden
+ERROR_OPENING_PROJECT=Ein Fehler trat auf beim öffnen des Projekts
+ERROR_OPENING_WORKSPACE=Ein Fehler trat auf beim öffnen der Workspace
+ERROR_FAILED_TO_PARSE_TRACE_LOG_FILE=Fehler beim Verarbeiten vom Trace-Log
+
+MSG_FILE_CONTENT_CHANGED=Dateiinhalt hat sich verändert.
+MSG_TAB_CONTENT_CHANGED=Tabinhalt hat sich verändert
+MSG_OK_BUTTON=OK-Knopf gedrückt
+MSG_OPEN_PROJECT=Öffne Projekt
+MSG_PROJECT_ALREADY_OPENED=Projekt ist schon in der Workspace
+MSG_SELECT_DIR=Ordner auswählen
+MSG_STARTING=Starte
+MSG_STARTING_DEBUGGER=Starte Debugger für
+MSG_TRY_OPEN_PROJECT=Versuche Projekt zu öffnen von
+DEBUGGING=debugge...
+RUNNING=läuft...
+
+QUESTION_CREATE_DIR=Der Zielpfad existiert nicht. Soll der Ordner angelegt werden?
+QUESTION_NEW_WORKSPACE=Soll eine neue Workspace anglegt werden oder die momentane benutzt werden?
```

Comment:
ein neuer

Reply:
Changed it. I think "verwendet" sounds better than "benutzt".